src/hotspots-plugin.tsx

```diff
@@ -147,13 +147,51 @@ export class HotspotsPlugin extends KalturaPlayer.core.BasePlugin {
     }));
   };
 
+  private _previousHotspotMap: Map<string, string> = new Map();
```

Comment:
why do we need to create this dynamically ?

Reply:
ok, if there's no other way, please add a comment that explains why you had to use dom manipulation because there's no other way to get the screen reader to read out the change